templates/rzp-pdp-checkout-btn.php

```diff
@@ -22,15 +22,23 @@ class="button alt single_add_to_cart_button"
 </div>
 
 <script type="">
-  var quantity = document.getElementsByClassName("qty")[0].value;
+  let i = 0;
+  while (typeof quantity === 'undefined') {
```

Comment:
why is a `while` loop needed? Won't buy now only have single item and quantity field?

Reply:
@varun-315 Some themes will not have the qty info on the 0th index so we need to loop it through to get the right qty selected.